tests/Fixer/LanguageConstruct/NoUnsetOnPropertyFixerTest.php

```diff
@@ -53,21 +53,28 @@ public function provideFixCases()
                 '<?php unset($foo->bar, $bar->foo, $bar->baz, $a->ba);',
             ],
             'It replaces multiple unsets, but not those that arent properties' => [
-                '<?php $foo->bar = null; $bar->foo = null; unset( $bar);',
+                '<?php $foo->bar = null; $bar->foo = null; unset($bar);',
                 '<?php unset($foo->bar, $bar->foo, $bar);',
             ],
             'It replaces multiple unsets, but not those that arent properties in multiple places' => [
-                '<?php  unset($foo); $bar->foo = null; unset( $bar);',
+                '<?php unset($foo); $bar->foo = null; unset($bar);',
                 '<?php unset($foo, $bar->foo, $bar);',
             ],
-            'It replaces multiple unsets, and does not mess with messy whitespace' => [
-                '<?php $foo->bar = null; $bar->foo = null; unset(   $bar );',
-                '<?php unset($foo->bar, $bar->foo,   $bar );',
-            ],
-            'It replaces $this->> and self:: replacements' => [
-                '<?php $this->bar = null; self::$foo = null; unset( $bar);',
+            'It replaces $this -> and self:: replacements' => [
+                '<?php $this->bar = null; self::$foo = null; unset($bar);',
                 '<?php unset($this->bar, self::$foo, $bar);',
             ],
+            'It does not replace unsets on arrays' => [
+                '<?php unset($bar->foo[0]);',
```

Comment:
add `'<?php unset($bar->foo{0});',` as well.
all hail php...

Reply:
I never knew you could do that, and now i wish i didn't know.

Reply:
One thing about writing fixers is you come across a lot of syntax you wish didn't exist 😆

Reply:
@BackEndTea , don't you worry. simply start using `normalize_index_brace` and forgot about that syntax in your life ;)
(well, just not in this PR ;P thanks for adding a test :+1:)

Reply:
you can also mix those up, so please add the test for `<?php unset($bar->foo{0}[1]{2}{$a}[$b]);`